packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -81,12 +103,23 @@
   // Move label when text-field gets autofilled in Chrome
   // stylelint-disable plugin/selector-bem-pattern
   &:-webkit-autofill + .mdc-floating-label {
-    transform: translateY(-100%) scale(.75);
+    transform: translateY(-50%) scale(.75);
     cursor: auto;
   }
   // stylelint-enable plugin/selector-bem-pattern
 }
 
+.mdc-floating-label {
```

Comment:
I'm pretty sure all of these styles except for `pointer-events: none;` are now redundant with styles in the mdc-floating-label package.

Also, there's lint complaints on this line.

```
 112:1  ✖  Invalid component selector ".mdc-floating-label"                                                                                  plugin/selector-bem-pattern
 112:1  ✖  Expected selector ".mdc-floating-label" to come before selector ".mdc-text-field__input:-webkit-autofill + .mdc-floating-label"   no-descending-specificity
```